### PR TITLE
Remove Gitlab CI config, stopping it running

### PR DESCRIPTION
**Pull request by stevejalim**
Now that everything from gitlab-ci.yml is replicated in Github Actions, we no longer need to trigger those jobs in Gitlab, so we're deleting the configuration file. 

Indeed, we no longer need to mirror to GL at all, so we're also dropping the GH action that performed that mirroring.

There will be a separate PR to www-config that trims that back to be just configuration, and no CI running.

### CODE REVIEW

**Review by alexgibson — APPROVED** [review]

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,101 +0,0 @@
...
-    CLUSTERS: iowa-a
-    NAMESPACE: bedrock-test
-
-prod:
-  extends: .update-config
-  only:
-    - prod
-  variables:
-    CLUSTERS: mozmeao-fr iowa-a
-    NAMESPACE: bedrock-prod
```
Comment (robhudson) [inline .gitlab-ci.yml:101]: 🎉 🎆 🥳

**Review by robhudson — APPROVED** [review]

**Comment by stevejalim** [comment]
Do not merge. I have a better plan

**Comment by stevejalim** [comment]
@pmac @bkochendorfer Just as a safety check: do both of you agree that we can unplug all GL behaviour for `bedrock`? There are still a couple of CI things on GL we need to move to GH, but those do not happen in gitlab.com/mozmeao/bedrock

**Review by pmac — APPROVED** [review]
Momentous occasion! Great work Steve!! 🥇

**Pull request merged**